### ISSUE TITLE
Issues with integers

### ISSUE THREAD

**Issue by dasbrow** [nicolaslopezj/searchable#171]
When I try to do searches where the query column is an integer I receive errors that I must use cast. I am not sure where to place the case if at all, even if I did it would cause another issue as two of my other search columns are strings. DB: postgresql, laravel 5.6

**Comment by nicolaslopezj** [comment]
Please make a pull request with the fix

**Comment by sachalegrand** [comment]
Yes. It seems there is no problem searching integer/bigint columns in MySQL or SQLite, but Postgresql will give an error.
